packages/at_secondary_server/lib/src/verb/handler/pkam_verb_handler.dart

```diff
@@ -144,4 +137,44 @@ class PkamVerbHandler extends AbstractVerbHandler {
       throw UnAuthenticatedException('pkam authentication failed');
     }
   }
+
+  @visibleForTesting
+  Future<Response> fetchApkamPublicKey(
+      String key, String enrollId, Response response) async {
+    var enrollData = await keyStore.get(key);
```

Comment:
keystore.get(key) throws 'KeyNotFoundException' when an enrollment key does not exist. 
Please handle the case when an enrollment key does not exist in the keystore.

Reply:
This case will be handled by the GlobalExceptionHandler